frontend/src/lib/utils/responsiveUtils.ts

```diff
@@ -1,5 +1,14 @@
 import { responsiveMap } from 'antd/lib/_util/responsiveObserve'
-import { ANTD_EXPAND_BUTTON_WIDTH } from './index'
+import { ANTD_EXPAND_BUTTON_WIDTH } from '../components/ResizableTable'
+
+const BREAKPOINT_MAP = Object.entries(responsiveMap).reduce<Record<string, number>>(
+    (acc, [key, cssStatement]) => ({
+        ...acc,
+        [key]: parsePixelValue(cssStatement),
+    }),
+    {}
+)
```

Comment:
Let's avoid using `reduce` whenever possible. The only thing using `reduce` shows to whoever is reading the code is that the author is proficient in using `reduce`, _nothing_ else :). It's usually longer and always less readable (`==` maintainable) than all other ways of doing the same thing.

The following does the same, but is much simpler to understand:

```js
const BREAKPOINT_MAP = Object.fromEntries(
    Object.entries(responsiveMap).map(([key, cssStatement]) => [key, parsePixelValue(cssStatement)])
)
```

... and is most likely much faster as well (no need to create and spread `array.length` temporary objects).

Reply:
super clean, thanks for the tip :)